README.md

```diff
@@ -2,8 +2,7 @@
 
 [![Test application](https://github.com/BonnierNews/eslint-config-exp/actions/workflows/run-tests.yml/badge.svg?branch=master)](https://github.com/BonnierNews/eslint-config-exp/actions/workflows/run-tests.yml)
 
-Basic [ESLint](https://eslint.org/) rules used by [Expressen](https://www.expressen.se). The test-config (`eslint-config-exp/test`) is adapted to
-testing using `mocha`, `mocha-cakes-2` and `chai`.
+Basic [ESLint](https://eslint.org/) rules used by [Expressen](https://www.expressen.se).
```

Comment:
Mention the auto-detection of modules vs common js and ES2022!

Reply:
I moved it up to the top 👌